src/components/Login.jsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useState } from 'react';
+import { Grid, Paper, Avatar, TextField, Button, Typography, Link } from '@mui/material';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import GoogleIcon from '@mui/icons-material/Google';
+import { useUserContext, signInWithGmail } from '../context/userContext';
+import '../App.css';
+
+const paperStyle = {
+  padding: 20,
+  height: '60vh',
+  width: '300px',
+  margin: '0 auto',
+};
+const avatarStyle = { backgroundColor: '#1bbd7e' };
+const btnStyle = { margin: '0.5rem 0' };
+
+// eslint-disable-next-line react/prop-types
+function Login({ handleChange }) {
+  const [loginEmail, setLoginEmail] = useState('');
+  const [loginPassword, setLoginPassword] = useState('');
+
+  const { loginUser, forgotPassword } = useUserContext();
+
+  const onSubmitHandler = (e) => {
+    e.preventDefault();
+    console.log(loginEmail);
+    console.log(loginPassword);
+    if (loginEmail && loginPassword) loginUser(loginEmail, loginPassword);
+  };
+
+  const handleForgotPassword = () => {
+    if (loginEmail) {
+      forgotPassword(loginEmail).then(() => setLoginEmail(''));
+    }
+  };
+
+  return (
+    <Grid>
+      <Paper style={paperStyle}>
+        <Grid align='center'>
+          <Avatar style={avatarStyle}>
+            <LockOutlinedIcon />
+          </Avatar>
+          <h2>Sign in</h2>
+        </Grid>
+        <form onSubmit={onSubmitHandler}>
+          <TextField
+            label='Email'
+            placeholder='Enter email'
+            fullWidth
+            required
+            variant='standard'
+            value={loginEmail}
+            onChange={(e) => {
+              setLoginEmail(e.target.value);
+            }}
+          />
+          <TextField
+            label='Password'
+            placeholder='Enter password'
+            fullWidth
+            required
+            type='password'
+            variant='standard'
+            value={loginPassword}
+            onChange={(e) => {
+              setLoginPassword(e.target.value);
+            }}
+          />
+          <FormControlLabel
+            control={<Checkbox name='checkedB' color='primary' />}
+            label='Remember me'
+          />
+          <Button type='submit' color='primary' fullWidth variant='contained' style={btnStyle}>
+            Sign in
+          </Button>
+          <Button
+            type='submit'
+            color='primary'
+            fullWidth
+            variant='contained'
+            style={btnStyle}
+            onClick={signInWithGmail}
+            sx={{ display: 'flex', justifyContent: 'center' }}
```

Comment:
Is this inline setting from material-ui?

Reply:
Yes, The sx prop is a shortcut for defining custom style that has access to the Material UI theme. So we can change the colour of the theme directly from the material ui theme with the sx prop.